### ISSUE TITLE
There is white blank area when lockToLandscape

### ISSUE THREAD

**Issue by muhammadwafa** [wonday/react-native-orientation-locker#106]
When I `lockToLandscape` a white blank space gets display which is not a good experience.
And also how to make a smooth transition when `orientation` changes

**Comment by muhammadwafa** [comment]
Wait for response ...

**Comment by romarybintsev** [comment]
Have you figured it out yet?

**Comment by RenanLopes77** [comment]
Any update ?

**Comment by romarybintsev** [comment]
I think I figured out the problem. It has to do with the header displaying for some reason after the orientation changes. Just add headerMode: 'none' to your stack navigator or to the specific screen where you call lockToLandscape

**Comment by muhammadwafa** [comment]
Thank you, There is no header for me

**Comment by wangchuxi** [comment]
@muhammadwafa  I also encountered this problem, did you solve it?

**Comment by wangchuxi** [comment]
@romarybintsev   Not that problem

**Comment by wangchuxi** [comment]
@romarybintsev thank you for head navigation
  Is caused by this head navigation of click and jump

Previous page
SettingsScreen.navigationOptions = ({ navigation, ...other }) => {
  // const info = navigation.getParam('pageInfo') || {};
  return {
    header: null,

**Comment by osikes** [comment]
headerMode: 'none' works. This could be closed I think.

**Comment by muhammadwafa** [comment]
I dont have any header, my header is null, and my app is without header

**Comment by Meycon** [comment]
Very strange, my Header is null and the white area continues ;/

**Comment by soyouwantme** [comment]
same issue, but none of them works for me.

**Comment by soyouwantme** [comment]
> same issue, but none of them works for me.

headerMode: 'none' works for me :P

**Comment by maximilize** [comment]
I don't think this issue is related to `react-navigation`, so `headerMode: 'none'` or `'screen'` will not work.

This issue happens even when using just a single `View` and switching to landscape on devices with a camera notch in the screen.

**Comment by manglesh-oodles** [comment]
> 

@maximilize  any solution for this issues?

**Comment by maximilize** [comment]
@manglesh-oodles With the lib https://www.npmjs.com/package/react-native-system-navigation-bar you have some more flexibility. We currently use both libs for our project.

**Comment by 7chenys** [comment]
May I ask if this problem has been solved? Now I have also encountered this problem

**Comment by 7chenys** [comment]
The above modification to HeaderMode: "none" doesn't work for me

**Comment by 7chenys** [comment]
> I don't think this issue is related to `react-navigation`, so `headerMode: 'none'` or `'screen'` will not work.
> 
> This issue happens even when using just a single `View` and switching to landscape on devices with a camera notch in the screen.

That's what I was thinking. I had the same problem
